releasenotes/notes/0.45/removed_deprecation_0.20-17b0ea6a3980eb4f.yaml

```diff
@@ -1,18 +1,17 @@
 ---
-upgrade:
+upgrade_visualization:
   - |
-    The class constructor arguments `qregs`, `cregs`, 
-    `layout` and `global_phase` for :class:`.visualization.QCircuitImage`
+    The class constructor arguments ``qregs``, ``cregs``,
+    ``layout`` and ``global_phase`` for :class:`.visualization.QCircuitImage`
     are removed, as they were deprecated in 0.20.
-    
+upgrade_transpiler:
   - |
-    In :class:`.transpiler.CouplingMap` method `subgraph` is removed
-    as deprecated in 0.20. :meth:`~reduce` can be used in place of method `subgraph`.
-  
+    In :class:`.transpiler.CouplingMap` method ``subgraph`` is removed
+    as deprecated in 0.20. :meth:`~reduce` can be used in place of method ``subgraph``.
+upgrade_synthesis:
   - |
-    The parameter `order` in :class:`.synthesis.SuzukiTrotter`
+    The parameter ``order`` in :class:`.synthesis.SuzukiTrotter`
     constructor raises an exception instead of deprecation warning when
     set in an odd number. Suzuki product formulae are symmetric and
     therefore only defined for even orders.
```

Comment:
Was there a reference to the original deprecation for this one?

Reply:
This is the original deprecation: https://github.com/Qiskit/qiskit/commit/29dddabb6fb5fd2821b6169e0365d24d7d78b6b4#diff-fca019ac849b4d2959c9b0303c5ea631f0d483ad2a6b64ed765891131203114cL54